hypnos/src/clock.c

```diff
@@ -73,25 +74,32 @@ void clock_print_time()
 	char mon[15];
 
 	if (sscanf(clock_get_local_time(), "%s %s", wday, mon) != 2) {
-		printk("Failed to print time.\n");
+		LOG_ERR("Failed to print time!");
 	}
-	printf("\n%d:%d:%d | ", localtime(&local_time)->tm_hour,
+	LOG_INF("%s %d %s\n", log_strdup(wday), localtime(&local_time)->tm_mday,
+		log_strdup(mon));
+	LOG_INF("%d:%d:%d | ", localtime(&local_time)->tm_hour,
```

Comment:
I guess we don't need the pipe characher since you've put date and time on separate lines.

Reply:
Fixed